### PR TITLE
ADB-all-in-1-template-4-vnet-injection-and-pvtlink

### PR DESCRIPTION
**Pull request by v-gsapaharam**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by v-gsapaharam** [comment]
Will review

**Review by msaranga — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-and-privatelink/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+    },
+    "publicSubnetName": {
+      "type": "string",
+      "defaultValue": "public-subnet",
+      "metadata": {
+        "description": "The name of the public subnet to create."
+      }
+    },
+	"requiredNsgRules": {
+		"defaultValue": "AllRules",
```
[the remark this answers is not in the document]
Comment (v-gsapaharam) [reply quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-and-privatelink/azuredeploy.json:82]: Updated default value : "NoAzureDatabricksRules"

File: quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-and-privatelink/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+      }
+    },
+    "privateSubnetName": {
+      "type": "string",
+      "defaultValue": "private-subnet",
+      "metadata": {
+        "description": "The name of the private subnet to create."
+      }
+    },
+	"publicNetworkAccess": {
```
[the remark this answers is not in the document]
Comment (v-gsapaharam) [reply quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-and-privatelink/azuredeploy.json:60]: Updated with Default value "Disabled"

File: quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-and-privatelink/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+          }
+        ]
+      },
+      "dependsOn": [
+        "[resourceId('Microsoft.Network/networkSecurityGroups', parameters('nsgName'))]"
+      ]
+    },
+    {
+      "type": "Microsoft.Databricks/workspaces",
+      "apiVersion": "2018-04-01",
```
[the remark this answers is not in the document]
Comment (v-gsapaharam) [reply quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-and-privatelink/azuredeploy.json:262]: Updated API-version.

File: quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-and-privatelink/GettingStarted.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+**Nsg Name** Name of the network Security Group
+**Vnet Name** Name of the Virtual Network
+**Private Subnet Name** Name of the Private Subnet
+**Public Subnet Name** Name of the Public Subnet
+**Location** Location of Data Center
+**Vnet Cidr** Cidr Range of the Vnet
+**Private Subnet Cidr** Cidr Range of the Private Subnet
+**Public Subnet Cidr** Cidr Range of the Public Subnet
+
```
[the remark this answers is not in the document]
Comment (v-gsapaharam) [reply quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-and-privatelink/GettingStarted.md:28]: Added respective properties

File: quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-and-privatelink/GettingStarted.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
+# Deploy a Network Security Group, a Virtual Network and an Azure Databricks Workspace with the Virtual Network
```
[the remark this answers is not in the document]
Comment (v-gsapaharam) [reply quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-and-privatelink/GettingStarted.md:1]: Updated the MD file

**Comment by shailja-khurana** [comment]
> Will review



> Will review

looks good

**Comment by satishraj777** [comment]
Hi

**Comment by SatishGandamuni** [comment]
Hi

**Comment by bmoore-msft** [comment]
Closing abandoned request

**Pull request closed without merging**